lib/uksched/include/uk/sched.h

```diff
@@ -98,6 +98,13 @@ struct uk_sched {
 	struct uk_alloc *a_auxstack; /**< default allocator for aux stacks */
 	struct uk_alloc *a_uktls; /**< default allocator for TLS+ectx */
 	struct uk_sched *next;
+#if CONFIG_LIBUKSCHED_STATS
+	__nsec start_time;
+	unsigned long sched_count;
+	unsigned long yield_count;
+	unsigned long next_count;
+	unsigned long idle_count;
```

Comment:
I am just seeing that we should also update the `uk_sched_init` macro so that the counters are reset. I am thinking to use the partial struct initialization because that makes sure that any new field in the future is automatically initialized with `0`:
```c
*(s) = (struct uk_sched){ \
    .thread_add = (thread_add_func);
    /* .. */
}
```